### DISCUSSION TITLE
Roadmap updates

### DISCUSSION THREAD

**Discussion by andrewleader** [microsoft/WindowsAppSDK#507] [Announcements]
UPDATE: We are now using our productboard portal to share roadmap updates. If you vote on a feature, you'll receive status updates when that feature changes status! See here for more info: https://github.com/microsoft/WindowsAppSDK/discussions/1778

OLD: Subscribe to this discussion to be notified of any roadmap updates/changes!

**Comment by andrewleader** [comment]
3/2/2021 - Unfortunately due to some work taking slightly longer than planned, the **unpackaged support that was previously planned won't be coming in 0.5 and has been pushed back to 0.8**. The work to complete unpackaged support just barely missed the dates to have it completed!

**Before for unpackaged apps** (only the 0.5 release was changed)

 🌟 |       Reunion 0.5 (Q1 2021)      |         Reunion 0.8 (Q2 2021)   |       Reunion 1.0 (Q4 2021)    | 
-- | ---------------------------------- | ------------------------------ | ------------------------------- |
WinUI |                ❌ Not Supported |              ❌ Not Supported |                   ✅ Supported  | 
DWriteCore |                 🔄 Preview |                    🔄 Preview |                   ✅ Supported  | 
MRT Core |                   🔄 Preview |                    🔄 Preview |                   ✅ Supported  | 
App Lifecycle APIs |         🔄 Preview |                    🔄 Preview |                   ✅ Supported  | 
Power State Notifications |❌ Not Supported |                🔄 Preview |                   ✅ Supported  | 
Windowing |                ❌ Not Supported |          ❌ Not Supported |                   ✅ Supported  | 
Deploy Options:  |  
• In-app |             ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  | 
• Fx Package |               🔄 Preview |                    🔄 Preview |                   ✅ Supported  | 


**Now for unpackaged apps** (only the 0.5 release was changed)

 🌟 |       Reunion 0.5 (Q1 2021)      |         Reunion 0.8 (Q2 2021)   |       Reunion 1.0 (Q4 2021)    | 
-- | ---------------------------------- | ------------------------------ | ------------------------------- |
WinUI |                ❌ Not Supported |              ❌ Not Supported |                   ✅ Supported  | 
DWriteCore |           ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  | 
MRT Core |             ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  | 
App Lifecycle APIs |   ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  | 
Power State Notifications |❌ Not Supported |                🔄 Preview |                   ✅ Supported  | 
Windowing |                ❌ Not Supported |          ❌ Not Supported |                   ✅ Supported  | 
Deploy Options:  |  
• In-app |             ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  | 
• Fx Package |         ❌ Not Supported |                    🔄 Preview |                   ✅ Supported  |

**Comment by andrewleader** [comment]
6/16/2021 - Push notifications has been added to the roadmap as Preview in the 1.0 timeframe!

**Reply by lighterhy** [reply]
Have you removed "WinUI 3 in UWP apps" from WinUI 3 roadmap? What is the plan for UWP apps？

**Comment by andrewleader** [comment]
9/14/2021 - Some roadmap changes...

* Self-contained (InApp/AppLocal) support has been moved out of 1.0 but will be supported in a future version
* Added single-project MSIX to the packaged roadmap
* Added elevation support to the roadmaps
* Added dynamic dependencies to the roadmaps
* Renamed "InApp" and "FxPackage" to "Self-contained" and "Framework-package" to match .NET terminology

**Comment by andrewleader** [comment]
We are now using our productboard portal to share roadmap updates. If you vote on a feature, you'll receive status updates when that feature changes status! See here for more info: https://github.com/microsoft/WindowsAppSDK/discussions/1778
